mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/clients/arenaadapter/ArenaAdapterClientImpl.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.mulighetsrommet.api.clients.arenaadapter
+
+import io.ktor.client.call.*
+import io.ktor.client.engine.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.plugins.*
+import io.ktor.client.plugins.cache.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import no.nav.mulighetsrommet.api.setup.http.httpJsonClient
+import no.nav.mulighetsrommet.domain.dto.ExchangeTiltaksnummerForIdResponse
+import no.nav.mulighetsrommet.secure_log.SecureLog
+import org.slf4j.LoggerFactory
+
+private val log = LoggerFactory.getLogger(ArenaAdapterClientImpl::class.java)
+
+class ArenaAdapterClientImpl(
+    private val baseUrl: String,
+    private val machineToMachineTokenClient: () -> String,
+    clientEngine: HttpClientEngine = CIO.create()
+) : ArenaAdaperClient {
+    val client = httpJsonClient(clientEngine).config {
+        install(HttpCache)
+    }
+
+    override suspend fun exchangeTiltaksnummerForUUID(tiltaksnummer: String): ExchangeTiltaksnummerForIdResponse? {
```

Comment:
Kanskje rename metode til `exchangeTiltaksnummerForId` så det samsvarer med responsklasse?

Reply:
Nå ble jeg usikker, var det ikke arena-id som skulle utveksles for å hente mulighetsrommet-id, og ikke tiltaksnummeret?

Reply:
Ah, er vel bare litt uheldig bruk av variabler/metodenavn her tror jeg. Ting kalles riktig i arena-adapter, men vi burde nok kalle det for `arenaId` i stedet for `tiltaksnummer`, slik at vi ikke blander det med `tiltaksnummer`-kolonnen vi lagrer på tiltaksgjennomføringer